lsp/src/Language/LSP/VFS.hs

```diff
@@ -458,15 +427,12 @@ codeUnitOffsetToCodePointOffset rope offset = do
  the position.
```

Comment:
Hmm, is this definitely still the same? For some reason I thought it might be faster :thinking: But maybe it's just that the code is simpler...

Reply:
It just remove the fromText and toText part, saving the conversion, other logic still the same.
since the mixed version have two indices, one for codePoint, one for utf16